types/gulp-postcss/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for gulp-postcss 8.0
+// Project: https://www.npmjs.com/package/gulp-postcss
+// Definitions by: Takesi Tokugawa <https://github.com/TokugawaTakesi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node"/>
+import Vinyl = require('vinyl');
+
+declare function GulpPostCss(plugins?: string[], options?: GulpPostCss.Options): NodeJS.ReadWriteStream;
+declare function GulpPostCss(callback?: (file: Vinyl) => { plugins?: string[], options?: GulpPostCss.Options }):
+    NodeJS.ReadWriteStream;
+
+declare namespace GulpPostCss {
+    interface Options {
+        parser?: Object;
```

Comment:
It could be any other library or custom function. Please give me the hint how to annotate unknown in advance library.

Reply:
If in doubt, make it `any`.